packages/components/src/FormatDate/FormatDate.tsx

```diff
@@ -21,13 +35,19 @@ export function FormatDate({ date: inputDate }: FormatDateProps) {
     dateObject = new Date(inputDate.year, inputDate.month - 1, inputDate.day);
   }
 
-  return <>{strFormatDate(dateObject)}</>;
+  return <>{strFormatDate(dateObject, showYear)}</>;
 }
 
-export function strFormatDate(date: Date) {
-  return date.toLocaleDateString(undefined, {
-    year: "numeric",
+export function strFormatDate(date: Date, showYear = true) {
+  let formatOptions: formatOptions = {
```

Comment:
```suggestion
  let formatOptions: Intl.DateTimeFormatOptions = {
```

Reply:
Good suggestion made the change [here](https://github.com/GetJobber/atlantis/pull/1301/commits/c382f2be8df523efafb2e64af73f710b9cc27291)